neps/nep-0480.md

```diff
@@ -0,0 +1,143 @@
+---
+NEP: 480
+Title: Account namespaces
+Authors: Jacob Lindahl <@encody>, Firat Sertgoz <@firatNEAR>
+Status: Draft
+DiscussionsTo: https://github.com/near/NEPs/pull/480
+Type: Protocol
+Version: 1.0.0
+Created: 2023-05-12
+LastUpdated: 2023-05-12
+---
+
+## Summary
+
+This proposal introduces account namespaces, which enable deploying multiple WASM binaries to a single account. A WASM binary is deployed to a unique, deployer-selected namespace. Subsequent deployments to the same namespace overwrite the code currently deployed to that namespace. The empty namespace is considered the default namespace.
+
+A WASM binary deployed to a non-default (non-empty) namespace can be invoked explicitly by specifying the namespace in the action. If no namespace is specified, the default namespace is used.
```

Comment:
```suggestion
A WASM binary deployed to a non-default (non-empty) namespace can be invoked explicitly by specifying the namespace in the action. If no namespace or the empty string is specified, then the default namespace is used.
```

Based on the information below in the NEP.

Reply:
That the empty namespace is the default namespace is specified elsewhere, but added for clarity.